.github/workflows/ci.yml

```diff
@@ -68,3 +68,30 @@ jobs:
 
       - name: Run tests
         run: ./vendor/bin/phpcs
+
+  static-analysis:
+    name: Static analysis
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
```

Comment:
is phpstan output really differs for all of these languages? I think we only can fix that one that refers to minimum php version we allow

Reply:
Yeah, I think we can just run it on one version. Which version should we run on?

Reply:
I prefer stick to lowest php version possible here in order not to accidently do checks for newer features

Reply:
Changed.